packages/react-native-bpk-component-button/src/BpkBorderedButton.ios.js

```diff
@@ -108,6 +109,10 @@ const BpkBorderedButton = (props: Props) => {
 
   return (
     <BpkTouchableOverlay
+      overlayStyle={{
+        borderColor: colorGray100,
+        borderWidth: buttonBorderWidth,
+      }}
```

Comment:
Can we include this in the `styles` object for consistency?

Reply:
Well we could, but `borderColor` is provided as a prop so we would have to set that anyway.

Reply:
Fair enough